AIMSICD/src/main/AndroidManifest.xml

```diff
@@ -1,56 +1,34 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+    <uses-permission android:name="android.permission.ACCESS_SUPERUSER"/>
     <uses-permission android:name="android.permission.CHANGE_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.INTERACT_ACROSS_USERS_FULL"/>
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
-    <uses-permission android:name="android.permission.INTERACT_ACROSS_USERS_FULL"/>
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
     <uses-permission android:name="android.permission.VIBRATE"/>
     <uses-permission android:name="android.permission.WAKE_LOCK"/>
-    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
 
     <!-- SPECIAL PERMISSIONS TO BE ADDED AFTER THIS LINE. /> -->
-    <!-- To list all available (used) Android permissions on a device, use:
-         # `pm list permissions -g`  -->
-
-    <!-- These are OEM / Samsung Permissions -->
-    <uses-permission android:name="android.phone.receiveDetailedCallState"/>
-    <uses-permission android:name="com.android.permission.HANDOVER_STATUS"/>
-    <uses-permission android:name="com.sec.android.app.controlpanel.permission.PRIVATE"/>
-    <uses-permission android:name="com.sec.android.app.factorymode.permission.KEYSTRING"/>
-    <uses-permission android:name="com.sec.android.app.cm.permission.PERMISSION_MANAGEMENT"/>
-    <uses-permission android:name="com.sec.android.app.phoneutil.permission.KEYSTRING"/>
-    <uses-permission android:name="com.sec.android.app.servicemodeapp.permission.KEYSTRING"/>
-    <uses-permission android:name="com.sec.android.phone.permission.DATA_ROAMING_SETTINGS_ENHANCED"/>
-    <uses-permission android:name="com.sec.android.phone.permission.READ_CALL_SETTINGS"/>
-    <uses-permission android:name="com.sec.android.phone.permission.WRITE_CALL_SETTINGS"/>
-    <!-- uses-permission android:name="com.sec.factory.permission.ALLOWFTCLIENTCPOBIND"/>
-    <uses-permission android:name="com.sec.factory.permission.BT_ID_WRITE"/ -->
-    <uses-permission android:name="com.sec.factory.permission.KEYSTRING"/>
-    <uses-permission android:name="com.sec.modem.settings.permission.KEYSTRING"/>
-    <!-- uses-permission android:name="diagandroid.app.receiveDetailedApplicationState"/>
-    <uses-permission android:name="diagandroid.data.receivePDPContextState"/>
-    <uses-permission android:name="diagandroid.phone.receiveDetailedCallState"/ -->
 
     <!-- May be needed (in the future) to access SIM related functions -->
     <uses-permission android:name="org.simalliance.openmobileapi.SMARTCARD"/>
```

Comment:
As far as I know our app does not access the SIM yet. But would it hurt letting this one enabled?